traildb/traildb.py

```diff
@@ -414,6 +415,42 @@ def create_filter(self, event_filter):
         return TrailDBEventFilter(self, event_filter)
 
 class TrailDBEventFilter(object):
+    """
+    Converts a query defined in terms of Python collections to a
+    `tdb_event_filter` which can be passed to various TrailDB functions.
+    Performs some validation when parsing the query.
+
+    Queries are boolean expressions defined from terms and clauses.  A term is
+    defined using a tuple:
+
+    (field_name, "value") -- match records with field_name == "value"
+    (field_name, "value", False) -- match records with field_name == "value"
+    (field_name, "value", True) -- match records with field_name != "value"
```

Comment:
Hmm this could be a bit confusing. Intuitively I could easily read `(field_name, "value", True)` as `field name equals value is True` or something like that, not obvious that `True` actually means "negative" :)

Maybe it should be `(field_name, "value", "eq")` or `(field_name, "eq", "value")` to make it less ambigious when reading code that uses the filters.